data/recommendation/localData.json

```diff
@@ -0,0 +1 @@
words","Usernames","Website activity"],"IsVerified":true,"IsSensitive":false,"IsActive":true,"IsRetired":false,"LogoType":"png"},{"Title":"Pixel Federation","Name":"PixelFederation","Domain":"pixelfederation.com","BreachDate":"2013-12-04","AddedDate":"2013-12-06T00:00:00Z","PwnCount":38108,"Description":"In December 2013, a <a href=\"http://www.cyberwarnews.info/2013/12/04/pixel-federation-hacked-38000-user-credentials-leaked\" target=\"_blank\">breach of the web-based game community based in Slovakia</a> exposed over 38,000 accounts which  were promptly posted online. The breach included email addresses and unsalted MD5 hashed passwords, many of which were easily converted back to plain text.","DataClasses":["Email addresses","Passwords"],"IsVerified":true,"IsSensitive":false,"IsActive":true,"IsRetired":false,"LogoType":"svg"},{"Title":"Plex","Name":"Plex","Domain":"plex.tv","BreachDate":"2015-07-02","AddedDate":"2016-02-08T01:35:48Z","PwnCount":327314,"Description":"In July 2015, the discusison forum for Plex media centre <a href=\"https://blog.plex.tv/2015/07/02/security-notice-forum-user-password-resets\" target=\"_blank\">was hacked and over 327k accounts exposed</a>. The IP.Board forum included IP addresses and passwords stored as salted hashes using a weak implementation enabling many to be rapidly cracked.","DataClasses":["Email addresses","IP addresses","Passwords","Usernames"],"IsVerified":true,"IsSensitive":false,"IsActive":true,"IsRetired":false,"LogoType":"svg"},{"Title":"Pokemon Creed","Name":"PokemonCreed","Domain":"pokemoncreed.net","BreachDate":"2014-08-08","AddedDate":"2014-08-10T00:03:59Z","PwnCount":116465,"Description":"In August 2014, the Pokemon RPG website <a href=\"http://pokemoncreed.net\" target=\"_blank\">Pokemon Creed</a> was hacked after a dispute with rival site, <a href=\"http://pkmndusk.in\" target=\"_blank\">Pokemon Dusk</a>. In a <a href=\"https://www.facebook.com/ramandeep.s.dehal/posts/749666358442465\" target=\"_blank\">post on Facebook</a>, &quot;Cruz Dusk&quot; announced the hack then pasted the dumped MySQL database on <a href=\"http://pkmndusk.in\" target=\"_blank\">pkmndusk.in</a>. The breached data included over 116k usernames, email addresses and plain text passwords.","DataClasses":["Email addresses","Genders","IP addresses","Passwords","Usernames","Website activity"],"IsVerified":true,"IsSensitive":false,"IsActive":true,"IsRetired":false,"LogoType":"svg"},{"Title":"PS3Hax","Name":"PS3Hax","Domain":"ps3hax.net","BreachDate":"2015-07-01","AddedDate":"2016-02-07T04:44:49Z","PwnCount":447410,"Description":"In approximately July 2015, the Sony Playstation hacks and mods forum known as <a href=\"http://www.ps3hax.net\" target=\"_blank\">PS3Hax</a> was hacked and more than 447k accounts were exposed. The vBulletin forum included IP addresses and passwords stored as salted hashes using a weak implementation enabling many to be rapidly cracked.","DataClasses":["Email addresses","IP addresses","Passwords","Use
```

Comment:
I'd prettify this, using some automatic tool. That way if it changes later, it is easier to read the git diff to see what changed.